src/SelectField/SelectField.jsx

```diff
@@ -20,9 +20,15 @@ function getStyles(props) {
 }
 
 const SelectField = React.createClass({
+  displayName: 'SelectField',
 
   propTypes: {
     /**
+     * opens DropDownMenu and triggers focus of TextField
+     */
+    autoFocus: React.propTypes.bool,
```

Comment:
> called like that in DropDownMenu i just passed it.

Except rather than pass it, its called `autoFocus`, which is different to `openImmediately`. To be consistent it should be `openImmediately` in both, no?